### ISSUE TITLE
Help text toggle

### ISSUE THREAD

**Issue by kevinwhoffman** [WordImpress/google-maps-builder#85]
Help text is great for first-time users, but it takes up a lot of real estate in the UI.

I'm suggesting a toggle in the form of a question mark icon that sits next to any field label that requires explanation. Clicking the icon would reveal the help text as it is now. This provides a good middle ground for new users who need help and experienced users who may want to refer back to help, but don't need it visible by default. Mockup below:

![maps-builder-help-toggle](https://cloud.githubusercontent.com/assets/1938671/9424870/4913d434-48ca-11e5-8df3-35efa65afa01.jpg)

Note that I've also rearranged the Create Marker section to improve clarity in the absence of help text.

**Comment by DevinWalker** [comment]
I agree, it's been on my todo. It really lessens the clutter indeed and your mockup just adds to that point.

**Comment by DevinWalker** [comment]
I just finished adding the base logic for creating CMB2 tooltips - now I'm going to port over the field descriptions.

**Comment by kevinwhoffman** [comment]
Not that it's a huge deal for performance, but I used the help icon from Dashicons which is bundled with wordpress so you shouldn't need to use an image. https://developer.wordpress.org/resource/dashicons/#editor-help

**Comment by DevinWalker** [comment]
I'm bumping this to 2.1 because I feel we can't rely on the current hackish way of creating tooltips with CMB2. I'm looking into creating a PR to CMB2 to take care of the issue: https://github.com/WebDevStudios/CMB2/issues/435

**Comment by mathetos** [comment]
Bumping to 2.2 only because 2.1 is becoming more of an urgent clean-up fix-up kind of release.

**Comment by nishitlangaliya** [comment]
@DevinWalker , @kevinwhoffman ,

As per suggestion I have converted descriptions tips into hint.css tooltip and made PR for the same. I have prepared how it works video.

https://www.useloom.com/share/3df2112a0379410eb459a5de3e991cd7

Thanks
